strongbox-web-core/src/main/java/org/carlspring/strongbox/domain/DirectoryListing.java

```diff
@@ -1,15 +1,30 @@
 package org.carlspring.strongbox.domain;
 
+import java.net.URL;
 import java.util.ArrayList;
 import java.util.List;
 
 public class DirectoryListing
 {
 
+    /**
+     * URL to the current directory (i.e. http://localhost.dev/api/browse/some/path)
+     */
+    private URL link;
```

Comment:
It would be more intuitive to call this `url`.

Reply:
I would leave it as-is since `link` is closer to hateoas' convention which is `links` (which needs to be an array of links which we don't need).